plot.py

```diff
@@ -0,0 +1,305 @@
+import os
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.markers import MarkerStyle
+from matplotlib.cm import get_cmap
+
+
```

Comment:
I like how you make a full file for plotting functions. It cleans up the algorithm